sdk/core/Azure.Core/src/GeoJson/GeoLineString.cs

```diff
@@ -17,7 +17,7 @@ namespace Azure.GeoJson
     ///     new GeoPosition(-122.108727, 47.649383),
     ///     new GeoPosition(-122.081538, 47.640846),
     ///     new GeoPosition(-122.078634, 47.576066),
-    ///     new GeoPosition(-122.112686, 47.578559)
+    ///     new GeoPosition(-122.112686, 47.578559),
```

Comment:
+1. I thought about mentioning it, but we never settled on it as a team. In fact, .editorconfig doesn't support it yet. I opened a bug on this a long time back and started working on it, but the .NET team never quite settled on several aspects and I moved on from it. Maybe worth revisiting.